gateway/src/main/java/de/tum/in/www1/artemis/security/jwt/TokenProvider.java

```diff
@@ -0,0 +1,112 @@
+package de.tum.in.www1.artemis.security.jwt;
```

Comment:
Similar as above:
Here is the diff: https://www.diffchecker.com/kOMSbyho
[TokenProvider.java]